FilePersistence/src/version_control/Commit.cpp

```diff
@@ -103,4 +103,88 @@ bool Commit::getFileContent(QString fileName, const char*& content, int& content
 		return false;
 }
 
+QStringList Commit::regexSearchNodeData(QString id) const
```

Comment:
Don't call this method `regex...` it's not using regular expressions. Call it `nodeLinesFromId`.